React/Views/RCTView.m

```diff
@@ -1380,6 +1380,70 @@ - (void)drawFocusRingMask
 #pragma mark - macOS Event Handler
 
 #if TARGET_OS_OSX
+- (void)resetCursorRects
+{
+  switch (self.cursor) {
+    case RCTCursorAuto:
+      if (self.focusable) {
+        [self addCursorRect:self.bounds cursor:[NSCursor pointingHandCursor]];
+      }
```

Comment:
I'm concerned about `auto` here since most buttons and other clickable items are `focusable`, but showing the hand over those is more appropriate on the web than for native applications.

Reply:
I'm porting a web app so and using enough electron based apps that I didn't even notice this. You're right, most native applications tend not to use pointing hand on hover of buttons. I dropped this case.